source/api_reference/index.html.md.erb

```diff
@@ -232,6 +232,8 @@ For example:
 | visa             | CREDIT | Visa             |
 | master-card      | DEBIT  | Mastercard       |
 | master-card      | CREDIT | Mastercard       |
+| maestro          | DEBIT  | Maestro          |
+| maestro          | CREDIT | Maestro          |
```

Comment:
I don't think there is such a thing as a Maestro credit card

Reply:
Certainly there isn't anything in the BIN ranges for one

Reply:
I didn't think that either but Wikipedia has confused me: https://en.wikipedia.org/wiki/Maestro_(debit_card)

> Maestro (stylized as maestro) is a multi-national credit card service ...

Reply:
@rhowe-gds is right, we need to remove Maestro Credit

Reply:
No problem, done. Thanks both 👍